### ISSUE TITLE
Improve EVG 11.13 and 11.8.4 to clarify that automatic issuance is permitted

### ISSUE THREAD

**Issue by CBonnell** [cabforum/servercert#467]
Based on the discussion of the 2023-11-16 validation-sc meeting, it was agreed that EVG 11.13 and 11.8.4 can be interpreted to not require manual due diligence checks and manual approvals of each Certificate Request. However, the language is very clear this is allowed, so the language in these sections should be reworked to make this more explicit.

**Comment by MadZeg** [comment]
I assume you mean: "However, the language is **not** very clear.." EVG 14.1.3 also includes language that indicates that manual approval is required for each Certificate Request. This language should also be reworked to allow for automatic issuance.

**Comment by CBonnell** [comment]
Yes, thank you for spotting the typo @MadZeg. I just corrected the description.

**Comment by CBonnell** [comment]
As discussed on the 2025-04-03 validation-sc meeting, @dougbeattie will ping Eva to see next steps on the previously discussed ballot.

@clintwilson and @BenWilson-Mozilla are interested in helping to push a ballot forward.

**Comment by dzacharo** [comment]
@clintwilson @BenWilson-Mozilla @dougbeattie pinging to see if anyone can volunteer to "own" this issue :)

**Comment by dzacharo** [comment]
Perhaps you can find more details in https://groups.google.com/a/groups.cabforum.org/g/servercert-wg/c/4ZXTcS59Jl4/m/fn2MBwdsAQAJ. There is an attachment in docx.

**Comment by BenWilson-Mozilla** [comment]
Here are the updated section numbers:
**3.2.8.4 - Pre-Authorized Certificate Approver** - we should delete the requirement that an agreement be written and signed.  Click-through agreements should be allowed. There are other potential barriers in the current language, including the required involvement of three roles and the concept of "EV Authority" exercised "with respect to each future EV Certificate Request"
**3.2.13 - Final Cross-correlation** - we should delete the mandatory second-person review - “CA MUST have a person who is not responsible for the collection of information review all of the information and documentation…”  It prevents ACME and expert system review equipped with automated discrepancy-detection, or risk scoring.  Replace “person” with “an independent process or function, such as an automated system, validation engine, or risk-detection algorithm." Also replace “review of all documentation" with “review of discrepancies and exceptions" to avoid reliance on narrative review. Remove "CA MUST obtain and document further explanation… from Applicant, Certificate Approver, Certificate Requester…" which is human-intensive.  Replace with "discrepancies can be resolved through automated checks, authoritative data sources, etc."
**5.2.4 - Roles requiring separation of duties** - can be replaced with the following, which could be moved to another place in the EVGs, if advisable:
The CA MUST enforce control procedures that ensure no single person or single system component can independently validate and authorize the issuance of an EV Certificate.
1. To achieve this objective, the CA MUST implement independent and functionally isolated controls such that:
(a) All required EV validation steps are performed and recorded by one or more controlled processes, systems, or persons acting within defined authorization boundaries; and
(b) Final authorization to issue an EV Certificate is performed by a separate and independently controlled function, system component, or authorized role that is not solely reliant on the output of any single validation actor or process.
2. Functions performing validation and issuance authorization MUST be designed so that:
(a) neither control can be overridden by the other;
(b) no single human or automated process can unilaterally complete both functions, and
(c) issuance can only proceed when both independent controls indicate that all EV requirements have been met.
3.  Automation is permitted using automated systems, automated workflows, or technical enforcement mechanisms provided that the independence, integrity, and auditability of such controls are maintained.
4.  Final Cross-Correlation and Due Diligence described in Section 3.2.2.13 MAY be performed by either of the independent control functions described above, or by an automated process providing equivalent independence and assurance.
5.  All controls implemented to satisfy this section MUST be documented with: descriptions for how separation of duties is enforced in both human and automated contexts; evidence demonstrating the independence of validation and issuance authorization functions; immutable or tamper-evident authorization records; and detailed logging of validation actions.
